openedx_events/learning/data.py

```diff
@@ -190,3 +190,29 @@ class CourseDiscussionConfigurationData:
     unit_level_visibility = attr.ib(type=bool, default=False)
     plugin_configuration = attr.ib(type=dict, default={})
     contexts = attr.ib(type=List[DiscussionTopicContext], factory=list)
+
+
+@attr.s(frozen=True)
+class GradeData:
+    """
+    Attributes defined for Open edX grades data object.
+
+    Arguments:
+        user (int): identifier of the grade to which the grade belongs.
+        course_key (CourseData): Identifier of the course to which the grade belongs.
+        course_edited_timestamp (datetime): date the grade was edited.
```

Comment:
`course_edited_timestamp` is misleading if it was the grade the one edited, don't you think?

Reply:
I found it confusing at the original model. https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/grades/models.py#L567
the grade is not what is being edited, but the content?

Reply:
is it relevant here then? :thinking:

Reply:
So please, can you change the attribute description?

Reply:
Oh yes, of course, that makes sense to me too,  I really appreciate your advice.